src/main/java/com/conveyal/gtfs/loader/Table.java

```diff
@@ -31,6 +30,7 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(Table.class);
     private static final Requirement[] requirementsForEditorFields = new Requirement[]{EDITOR, REQUIRED, OPTIONAL};
+    private static final Requirement[] requirementsForSpecFields = new Requirement[]{REQUIRED, OPTIONAL};
```

Comment:
It's not clear what an "editor field" or "spec field" is. Please add documentation.

Reply:
This will be removed in favor of the for loop approach you advocated.

Reply:
Just to clarify: a key part of that for loop approach was the inlined boolean expression instead of referencing an array of allowed types. In this case I think just inlining the few boolean tests is more readable.